src/arrays.js

```diff
@@ -1,61 +1,157 @@
 const getNthElement = (index, array) => {
-  // your code here
+  const maxArrayindex = (array.length - 1);
+  // console.log(maxArrayindex);
+  const indexGiven = index;
+  // console.log(indexGiven);
+
+  if (indexGiven > maxArrayindex) {
+    const newIndex = indexGiven - 4;
+    // console.log('here');
+    return array[newIndex];
+  }
+  return array[indexGiven];
 };
 
 const arrayToCSVString = (array) => {
-  // your code here
+  const csvArray = array.toString();
+  return csvArray;
 };
 
 const csvStringToArray = (string) => {
-  // your code here
+  const array = string.split(',');
+  return array;
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  if (element !== '') {
+    const array2 = array.concat(element);
+    return array2;
+  }
+  return array;
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index, 1);
 };
 
 const numbersToStrings = (numbers) => {
-  // your code here
+  // const numberStringArray = [];
+  // for (let i = 0; i < numbers.length; i++) {
+  //   numberStringArray.push(numbers[i].toString());
+  // }
+  // return numberStringArray;
+
+  function turnIntoString(numberasnumber) {
+    return numberasnumber.toString();
+  }
+  return numbers.map(turnIntoString);
 };
 
 const uppercaseWordsInArray = (strings) => {
-  // your code here
+  // const upperCaseArray = [];
+  // for (let i = 0; i < strings.length; i++) {
+  //   upperCaseArray.push(strings[i].toUpperCase());
+  // }
+  // return upperCaseArray;
+
+  function toUpperCase(lowerCaseString) {
+    return lowerCaseString.toUpperCase();
+  }
+  return strings.map(toUpperCase);
 };
 
 const reverseWordsInArray = (strings) => {
-  // your code here
+  const reversedStrings = [];
+  for (let i = 0; i < strings.length; i++) {
+    const currentString = strings[i];
+    let reverseString = '';
+    for (let j = currentString.length - 1; j >= 0; j--) {
+      reverseString += currentString[j];
+    }
+    reversedStrings.push(reverseString);
+  }
+  return reversedStrings;
 };
 
 const onlyEven = (numbers) => {
-  // your code here
+  const evens = [];
+  return numbers.filter((element) => {
+    if (element % 2 !== 0) {
+      return (element[0]);
+    }
+    return evens;
+
+    
+  });
 };
 
 const removeNthElement2 = (index, array) => {
-  // your code here
+  const newArray = array.slice(0, index).concat(array.slice(index + 1, array.length));
```

Comment:
you can use `.splice` method on array prototype

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice